src/patternfly/components/Check/check.scss

```
@@ -11,6 +11,11 @@
   --pf-c-check__description--Color: var(--pf-global--Color--200);
   --pf-c-check__body--MarginTop: var(--pf-global--spacer--sm);
 
+  // Required checkbox
+  --pf-c-form__label-required--MarginLeft: var(--pf-global--spacer--xs);
```

just a nit - these var names should match the elements they apply to.

```suggestion
  --pf-c-check__label-required--MarginLeft: var(--pf-global--spacer--xs);
```

---

Thank you, should have caught this.